kontrakter/politi/bestilltransport/1.0/bestillingAvTransport.schema.json

```diff
@@ -0,0 +1,905 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://politiet.no/politi/fengsling/1.0/bestillingAvTransport",
+  "description": "Bestilling av transport i forbindelse med varetektsfengsling. Schema versjon 2023-6",
+  "type": "object",
+  "properties": {
+    "melding": {
+      "type": "string",
+      "const": "BESTILLING_TRANSPORT",
+      "description": "Navnet på meldingen. Enkelt å se på en melding hvilken type det er."
+    },
+    "forsendelse": {
+      "$ref": "#/definitions/forsendelse",
+      "description": "avsender og mottaker"
+    },
+    "hovedStraffesaksnummer": {
+      "$ref": "#/definitions/straffesaksnummer",
+      "description": "BL Saksbehandlersak er hovedsaken i et sakskompleks (som er 1 eller flere straffesaker)"
+    },
+    "bestillingAvTransportId": {
+      "$ref": "#/definitions/unikId",
+      "description": "Nøkkel på bestillingen, alle svar på bestilling skal koble til den samme nøkkelen."
+    },
+    "kontaktPersonKDI": { "$ref": "#/definitions/ansattPerson" },
+    "varetektSyklusId": {
+      "$ref": "#/definitions/unikId",
+      "description": "Til å begynne med vil dette være et løpenummer, kan endres til GUID senere. "
+    },
+    "personVaretektInfo": { "$ref": "#/definitions/personVaretektInfo" },
+    "restriksjonIsolasjon": { "$ref": "#/definitions/restriksjonIsolasjon" },
+    "paagrepetTidspunkt": { "type": "string", "format": "date-time" },
+    "paagrepetSted": {
+      "type": "string",
+      "description": "Beskriver hvor pågripelsen fant sted. For bostedsløse er det nødvendig å kontakte NAV-kontoret i denne kommunen."
+    },
+    "bestillingAvTransportDetaljer": {
+      "typeFremstilling": {
+        "description": "Indikerer formålet med transporten. I første omgang støttes transport direkte til varetekt. På sikt kan det lages støtte for fremstilling til rettsmøter generelt m.m.",
```

Comment:
Observasjon: `Description`-feltet brukes til både _"beskrivelse av hva feltet representerer"_, samt **kommentar**. 
Jeg ser den praktiske nytteverdien her og det er kjekt nå. Vi må uansett være klar over at andre del av denne beskrivelsen ("I første omgang støttes...") potensielt vil måtte fjernes på et eller annet tidspunkt for å ikke være utdatert og forvirrende. Kunne potensielt ha kommet rundt det ved å for eksempel endre ordlyden til "I versjon 1.0 av dette skjema støttes..", da vet vi at det gjelder versjon 1.0 og ikke nødvendigvis stemmer i versjon 1.1 om ikke beskrivelsen blir endret.